src/platform/user/authentication/constants.js

```diff
@@ -67,6 +69,12 @@ export const EXTERNAL_REDIRECTS = {
     : EXTERNAL_LINKS.MY_VA_HEALTH.STAGING,
   [EXTERNAL_APPS.MHV]: EXTERNAL_LINKS.MHV,
   [EXTERNAL_APPS.EBENEFITS]: EXTERNAL_LINKS.EBENEFITS,
+  [EXTERNAL_APPS.VA_FLAGSHIP_MOBILE]: `https://${
+    eauthEnvironmentPrefixes[environment.BUILDTYPE]
+  }fed.eauth.va.gov/oauthe/sps/oauth/oauth20/authorize`,
```

Comment:
Have you verified with Flagship Mobile follows the same environment setup `int.fed | pint.fed | sqa.fed | fed`

Reply:
They do use them, but I think not local or dev. That said, I think we still use them in our links for those lower envs since the eauth page will display the error that it does not recognize the client_id . But we can still use that to verify we are doing the right thing on out end since everything past that is out of our hands